alica_tests/src/test/test_alica_simple_plan_tree_serialization.cpp

```diff
@@ -36,10 +36,13 @@ class AlicaSerializationTest : public AlicaTestFixture
 
     void waitUntilPlan(long planId)
     {
-        alica::AlicaTime sleepTime = alica::AlicaTime::seconds(1);
-        do {
-            ae->getAlicaClock().sleep(sleepTime);
-        } while (!alica::test::Util::isPlanActive(ae, planId));
+        // Wait 5 seconds max
+        for (int i = 0; i < 5; i++) {
+            ae->getAlicaClock().sleep(alica::AlicaTime::seconds(1));
+            if (alica::test::Util::isPlanActive(ae, planId)) {
+                break;
+            }
+        }
```

Comment:
can add an assert here to check if the plan is indeed active, if not the tc should fail